docs/DeveloperGuide.md

```diff
@@ -270,58 +270,115 @@ _{Explain here how the data archiving feature will be implemented}_
-| `* *`    | user                                       | hide private contact details   | minimize chance of someone else seeing them by accident                |
-| `*`      | user with many persons in the address book | sort persons by name           | locate a person easily                                                 |
+| Priority | As a …​                                    | I want to …​                     | So that I can…​                                     |
+| -------- | ------------------------------------------ | ------------------------------ |-----------------------------------------------------|
+| `* * *`  | potential user exploring the app           | see the app populated with sample openings         | easily see how the app will look when it is in use. |
+| `* * *`  | user                                       | add an opening regarding the internship I am interested in               | keep note of it                                     |
+| `* * *`  | user                                       | delete an opening      | remove unwanted openings                            |
+| `* * *`  | user                                       | Be able to see all applications          | track my progress                                   |
+| `* *`    | user                                       | see submission deadlines   | check them                                          |
+| `*`      | user | be able to see the total number of accepted applications           | celebrate my success                              |
+| '*' | user | see all the outcomes of my application | properly assess my options |
 
 *{More to be added}*
 
 ### Use cases
 
-(For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
+(For all use cases below, the **System** is `Ultron` and the **Actor** is the `user`, unless specified otherwise)
 
-**Use case: Delete a person**
+**Use case:  UC 1 - Adding an opening**
 
 **MSS**
 
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
+1. User enters a command to add an opening
+2. Ultron adds the opening to its list of openings
+   Use case ends.
 
-    Use case ends.
 
 **Extensions**
 
-* 2a. The list is empty.
+1a. Ultron detects an error in the entered data.
+    1a1. Ultron informs the user that the data is invalid
+    Use case ends
+
+**Use case:  UC 2 - Deleting an opening**
+
+**MSS**
+
+1. User enters a command to delete an opening
+2. Ultron deletes the opening from its list of openings
+   Use case ends.
+
+
+**Extensions**
+
+1a. Ultron detects an error in the entered data.
+    1a1. Ultron informs the user that the data is invalid
+    Use case ends
+
+**Use case:  UC 3 - Editing an opening**
+
+**MSS**
+
+1. User enters a command to edit an opening
```

Comment:
Perhaps change lines like this (applies to other use cases) to "User enters opening with the details to change it."?
Can avoid using "command"

Reply:
User selects opening and includes updated details